### PR TITLE
Fill in model parameter details in Colab cloud training section

### PR DESCRIPTION
**Pull request by ehurturk**
Updated colab.rst by replacing the TODOs in the Training in the Cloud section with explanations for model parameters (epochs, architecture, latency_samples, ignore_checks, etc.).

### CODE REVIEW

File: docs/source/tutorials/colab.rst [hunk 1]
```diff
@@ -72,7 +72,24 @@ button and everything will finish in about 10 minutes.
 However, there are a lot of options below that you can use to tweak the training
 that are worth getting familiar with.
 
-TODO: explain the options.
+Here are the options explained:
+
+**Training Parameters**:
+* ``epochs``: Number of iterations for training. Higher number of epochs may result in better results, may also lead to *overfitting*.
```
Comment (sdatkinson) [inline docs/source/tutorials/colab.rst:78]: Overfitting doesn't happen with higher epochs. NAM uses basic validation set checkpointing and goes back to the best checkpoint from the whole training run after it finishes.

File: docs/source/tutorials/colab.rst [hunk 2]
```diff
@@ -72,7 +72,24 @@ button and everything will finish in about 10 minutes.
...
+* ``architecture``: Determines the model architecure. Options are: "standard" | "light" | "feather" | "nano", defaults to "standard". `standard` is the most complicated model, yet `nano` is the simplest model.
```
Comment (sdatkinson) [inline docs/source/tutorials/colab.rst:79]: I don't like listing the architecture options hard-coded here because it's not DRY.

Instead of "Options are:..." change to "Listed in order of decreasing CPU usage and accuracy." which will remain true.

File: docs/source/tutorials/colab.rst [hunk 3]
```diff
@@ -72,7 +72,24 @@ button and everything will finish in about 10 minutes.
...
+* ``latency_samples``: How far the output lags the input due to round-trip latency during reamping, in samples. Options are: "auto" | [int value of reamping latency].
+* ``ignore_checks``: Ignores the data quality checks and YOLOs it :)
+
+**User Metadata**:
+* ``use_metadata``: If enabled .nam file includes user metadata.
+* ``name``: Model name 
+* ``modeled_by``: Author name
+* ``gear_make``: Used gear make
+* ``gear_model``: Used gear model
+* ``gear_type``: Used gear tye. Options are "amp" | "pedal" | "pedal_amp" | "amp_cab" | "amp_pedal_cab" | "preamp" | "studio".
```
Comment (sdatkinson) [inline docs/source/tutorials/colab.rst:89]: Same, don't hard-code the enum values (not DRY).

File: docs/source/tutorials/colab.rst [hunk 4]
```diff
@@ -72,7 +72,24 @@ button and everything will finish in about 10 minutes.
...
+* ``tone_type``: Reamped tone type. Options are "clean" | "overdrive" | "crunch" | "hi_gain" | "fuzz".
```
Comment (sdatkinson) [inline docs/source/tutorials/colab.rst:90]: (DRY)

**Review by sdatkinson — CHANGES_REQUESTED** [review]
Thanks for the PR! A few tweaks to please incorporate

File: docs/source/tutorials/colab.rst (hunk 4 above)
Comment (ehurturk) [reply docs/source/tutorials/colab.rst:90]: Thanks for your suggestions! I really like this project and wanted to contribute in any way I could, sorry for the mistakes:)

**Review by sdatkinson — APPROVED** [review]
Thanks!
